swift_browser_ui/request/db.py

```diff
@@ -2,77 +2,60 @@
 
 
 import logging
-import random
-import asyncio
 import os
 import typing
+import asyncio
+import random
 
 import asyncpg
-import aiohttp.web
 
 
 MODULE_LOGGER = logging.getLogger("db")
 MODULE_LOGGER.setLevel(os.environ.get("LOG_LEVEL", "INFO"))
 
 
-def handle_dropped_connection(request: aiohttp.web.Request) -> None:
-    """Handle dropped database connection."""
-    MODULE_LOGGER.log(logging.ERROR, "Lost database connection, reconnecting...")
-    request.app["db_conn"].erase()
-    asyncio.ensure_future(request.app["db_conn"].open())
-    raise aiohttp.web.HTTPServiceUnavailable(reason="No database connection.")
-
-
 class DBConn:
     """Class for handling sharing request database connection."""
 
     def __init__(self) -> None:
         """."""
         self.log = MODULE_LOGGER
-        self.conn: asyncpg.connection.Connection = None
+        self.pool: asyncpg.Pool = None
 
     async def open(self) -> None:
         """Gracefully open the database."""
-        while self.conn is None:
+        while self.pool is None:
             try:
-                self.conn = await asyncpg.connect(
+                self.pool = await asyncpg.create_pool(
```

Comment:
other parameters that can be used (values are as an example):
```
max_queries=50000,
timeout=120,
command_timeout=180,
max_cached_statement_lifetime=0,
max_inactive_connection_lifetime=180
```

it might be a good idea to initializing with 0 connections allows the web server to start and also continue to live so `min_size=0`